parm/config/gfs/config.wavepostsbs

```diff
@@ -17,7 +17,7 @@ export DOIBP_WAV='NO' # Input boundary points
 export DOFLD_WAV='YES' # Field data
 export DOPNT_WAV='YES' # Station data
 export DOGRB_WAV='YES' # Create grib2 files
-if [[ -z ${waveinterpGRD} ]]; then
+if [[ -n "${waveinterpGRD}" ]]; then
```

Comment:
Where else is the variable `waveinterpGRD` used?

If we just do this here:
```
if [[ -n "${waveinterpGRD:-}" ]]; then
```

If it is only here, then there is no need to define empty strings in `config.wave`.
Line 89 in `config.wave` is also meaningless as the proper value for `waveinterpGRD` is being set in the `case` block above.

Reply:
waveinterpGRD is used in multiple scripts, for example a quick grep shows other places this is used: 

scripts/exgfs_wave_post_gridded_sbs.sh:72:  export waveinterpGRD=${waveinterpGRD?Var wavepostGRD Not Set}
scripts/exgfs_wave_post_gridded_sbs.sh:81:  echo "   Interpolated grids : $waveinterpGRD"
scripts/exgfs_wave_post_gridded_sbs.sh:105:  for grdID in ${waveGRD} ${wavepostGRD} ${waveinterpGRD}; do
scripts/exgfs_wave_post_gridded_sbs.sh:116:  for grdID in ${waveGRD} ${wavepostGRD} ${waveinterpGRD}; do
scripts/exgfs_wave_post_gridded_sbs.sh:140:    for intGRD in $waveinterpGRD
scripts/exgfs_wave_post_gridded_sbs.sh:169:    for grbGRD in $waveinterpGRD $wavepostGRD
scripts/exgfs_wave_post_gridded_sbs.sh:279:        for grdID in $waveinterpGRD
scripts/exgfs_wave_init.sh:82:  array=($WAVECUR_FID $WAVEICE_FID $WAVEWND_FID $waveuoutpGRD $waveGRD $waveesmfGRD $wavepostGRD $waveinterpGRD)

Reply:
Thanks @JessicaMeixner-NOAA 
Its (`waveinterpGRD`) use in `scripts/exgfs_wave_post_gridded_sbs.sh` is handled by the change in `config.wavepostsbs` as that config file is sourced in the job JGLOBAL_WAVE_POST_SBS.

Its use in `scripts/exgfs_waveinit.sh` should be handled as:
```
 array=($WAVECUR_FID $WAVEICE_FID $WAVEWND_FID $waveuoutpGRD $waveGRD $waveesmfGRD $wavepostGRD ${waveinterpGRD:-})
```
Infact, looking at this, all these variables should have a default `:-` and the lines after the case block in `config.base` are no longer relevant or necessary.

Reply:
Noted, perhaps we can open a separate issue to address these concerns as I feel like these are out of the scope of this PR which is to simply fix the bug reported in issue #1658 which I believe I have proved the update here does through answering questions and the tests.

Reply:
Feel free to open an issue and assign yourself if you would be comfortable improving the state of the wave configuration and use in the workflow.  If not, leave it open as a reminder of the issue and we will address it as resources permits.